README.md

```diff
@@ -6,8 +6,18 @@ A list of friendly active repos for contributing!
 
 ---
 
+# open-source
+How to get started with open source: https://www.youtube.com/watch?v=MkaIrwOlP6Y
+
+
+# defaang
+defaang / dojo clan Discord (in case people want to ask questions or chat about open source): https://discord.com/invite/nNtVfKddDD
+---
```

Comment:
You might need to add a blank line before this line for the horizontal line to show up correctly